docs/src/pages/components/chatbot/usage.angular.mdx

```diff
@@ -0,0 +1,33 @@
+_app.module.ts_
+
+```diff
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { AppComponent } from './app.component';
+- import { AmplifyAngularModule, AmplifyService } from 'aws-amplify-angular';
++ import { AmplifyUIAngularModule } from '@aws-amplify/ui-angular';
```

Comment:
@wlee221 I thought you also made the chat bot available under a `/legacy` path?

Reply:
Right, this should be


```suggestion
+ import { LegacyAmplifyUiModule } from '@aws-amplify/ui-angular/legacy';
```

Reply:
Same on line 17 as well!

Reply:
Good call, I just copied over the content forgot to update those.